learn/guides/asyncapi-tool.md

```diff
@@ -16,144 +16,147 @@ The Ballerina AsyncAPI tool will make it easy for you to start the development o
+
+## Execute the Generated Sources
+
+In order to execute the generated Ballerina sources, please follow the below steps,
+
+1. Go inside the folder where the generated files exist
+2. Execute the command `bal init` - this will generate a Ballerina.toml file
+3. Create a new ballerina file inside the folder (Ex: slack_service.bal ) and use this piece of code
 
 ```ballerina
 listener Listener webhookListener = new(8090);
 
 service AppService on webhookListener {
-  remote function onAppMention(GenericEventWrapper event) returns error? {
-      //Implement the logic to use the received `event` here.
-  }
+   remote function onAppMention(GenericEventWrapper event) returns error? {
+       //implement the logic to use the received `event` here.
+   }
 }
 ```
 
-4\. Execute the command below from the root of the folder to execute this service.
-
-```bash
-bal run
-```
+That’s it, You can execute the command `bal run` to execute this.
 
 Below are some example libraries generated using the tool.
 
 | Module     | AsyncAPI specification                                                                                                                | Generated and modified code                                                                                              | Published module                                                                               |
 | ---------- | ------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------- |
 | **Slack**  | <a href="https://github.com/ballerina-platform/asyncapi-triggers/blob/main/asyncapi/slack/asyncapi.yml">`asyncapi.yml` of Slack</a>   | <a href="https://github.com/ballerina-platform/asyncapi-triggers/tree/main/asyncapi/slack">asyncapi-triggers/slack</a>   | <a href="https://central.ballerina.io/ballerinax/trigger.slack">ballerinax/trigger.slack</a>   |
-| **GitHub** | <a href="https://github.com/ballerina-platform/asyncapi-triggers/blob/main/asyncapi/github/asyncapi.yml">`asyncapi.yml` of GitHub</a> | <a href="https://github.com/ballerina-platform/asyncapi-triggers/tree/main/asyncapi/github">asyncapi-triggers/github</a> | <a href="https://central.ballerina.io/ballerinax/trigger.github">ballerinax/trigger.github</a> |
+| **Twilio** | <a href="https://github.com/ballerina-platform/asyncapi-triggers/blob/main/asyncapi/twilio/asyncapi.yml">`asyncapi.yml` of Twilio</a> | <a href="https://github.com/ballerina-platform/asyncapi-triggers/tree/main/asyncapi/twilio">asyncapi-triggers/github</a> | <a href="https://central.ballerina.io/ballerinax/trigger.twilio">ballerinax/trigger.github</a> |
```

Comment:
```suggestion
| **Twilio** | <a href="https://github.com/ballerina-platform/asyncapi-triggers/blob/main/asyncapi/twilio/asyncapi.yml">`asyncapi.yml` of Twilio</a> | <a href="https://github.com/ballerina-platform/asyncapi-triggers/tree/main/asyncapi/twilio">`asyncapi-triggers/github`</a> | <a href="https://central.ballerina.io/ballerinax/trigger.twilio">`ballerinax/trigger.github`</a> |
```

Please add all code segments, keywords, filenames, etc. within backticks in the table.